proposals/json-td/index.html

```diff
@@ -71,37 +79,59 @@
       "type": "boolean,
       "description": "Whether the lamp is turned on",
       "href": "/things/lamp/properties/on"
+    },
+    "lightLevel" :
+    {
+      "type": "integer",
+      "description": "The level of light from 0-100",
+      "min" : 0,
+      "max" : 100,
+      "href": "/things/lamp/properties/lightLevel"
     }
   },
   "actions": {
     "toggle": {
+      "type" : "boolean",
       "description": "Toggle the lamp on and off"
+    },
+    "dim": {
+        "type" : "integer",
+        "min" : 0,
+        "max" : 100,
+        "description": "Change the level of light"
```

Comment:
Why would this require an action rather than just setting the lightLevel property? I realise you're trying to provide an example which includes the other properties of a Value object, but I'd suggest either removing these or coming up with a better example. Alternatively we could keep this example simple and consistent with the Thing Description spec and add separate action examples in the "Action object" section.